README.md

```diff
@@ -35,7 +35,25 @@
 > npm i libp2p-websockets
 ```
 
-### Example
+### Constructor properties
```

Comment:
Perhaps this is a good time to just replace this example with how to use it in a libp2p configuration with the passed options. That's likely much more useful than the existing examples as most people won't use this standalone.